uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-entitlements.html

```diff
@@ -1 +1,154 @@
-Entitlements
+<section class="panel panel-default panel-default-plain">
+  <header class="panel-heading">
+    <h2 class="panel-title">Contract Entitlements</h2>
+    <hr/>
+  </header>
+  <div class="table-responsive">
+    <table class="table">
+      <tr>
+        <td>
+          <strong>Position</strong>
+        </td>
+        <td>
```

Comment:
I would prefer to do it in HTML itself, but only for width as it is easier to see and control
`style="width: n%"`